test/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "bedrock-web-vc-store-test",
+  "version": "0.0.1-0",
+  "private": true,
+  "scripts": {
+    "test": "npm run test-node",
+    "test-node": "cross-env NODE_ENV=test mocha -r esm --preserve-symlinks -t 30000 -A -R ${REPORTER:-spec} --require test-mocha.js 10-api.js",
+    "coverage": "cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text-summary npm run test-node",
+    "coverage-ci": "cross-env NODE_ENV=test nyc --reporter=lcovonly npm run test-node",
+    "coverage-report": "nyc report",
+    "lint": "eslint ."
+  },
+  "dependencies": {
+    "bedrock": "^4.1.1",
```

Comment:
Currently this isn't testing through bedrock/bedrock-express, but through the internal helper.  Could remove all these other bedrock deps.  (And also the eslint ones, see above comment.)

Reply:
I have to have this because the main package.json uses these as `peerDependencies` so it needs to have these install in the test suite or it will not be able to find the modules.

Reply:
This is a good observation, but why is this the case.  My suggestion would be that to test this module properly would involved setting up a mock endpoint by way of bedrock-express in `test.js` and running tests using `bedrock-test` against that endpoint.  Is there something that makes this approach difficult?

Reply:
Here's something like what I'm suggesting.  If I understand correctly, the middleware for this browser fix should be attached to any mock endpoint we create via this mechanism.

https://github.com/digitalbazaar/bedrock-session-http/blob/master/test/test.js#L14

Reply:
Re the deps, I was mistakenly thinking the test was pulling in the helper function directly.  If that was done, it would avoid the bedrock api calls and probably just have missing dep warnings during install.  A real test setup as suggested would be nice if it's not too difficult.

In any case, I think the monogdb dep can be removed?